src/civic/index.js

```diff
@@ -649,64 +667,67 @@ const upload = async ({
         let mappedCount = 0;
         const postupload = [];
 
-        // resolve combinations
+        // Nested loop on recordList. Resolve combinations
```

Comment:
Try to keep comments more about what it is doing and why vs how it is implemented (unless it is particularly confusing). In this case it should be obvious to the dev this is nested just from the indenting so I would remove this. You could expand it to say something more useful perhaps like "Resolve combinations: splits civic evidence items drugs into separate evidence items based on their combination type. Splits variants into a list to indicate separate evidence items when variants have been linked as "or"

Reply:
ok. I put both comments above their respective code section.